### ISSUE TITLE
fixup link to threatdragon.github.io/threat-dragon/

### ISSUE THREAD

**Issue by jgadsden** [OWASP/threat-dragon#64]
**Describe the bug**
Link to https://threatdragon.github.io/threat-dragon/ does not work

**Expected behavior**
The original repo had link to http://mike-goodwin.github.io/owasp-threat-dragon/
The repo migrated to OWASP area should do the same, except for threatdragon.github.io/threat-dragon

**Environment**
 - Version: 1.3.1
 - Platform: Web App
 - OS: all
 - Browser: all

**To Reproduce**
* navigate to https://github.com/OWASP/threat-dragon
* click on link to left of screen in the About pane

**Any additional context, screenshots, etc**

**Comment by jgadsden** [comment]
For the moment have replaced link with the docs page, as it looks bad to have a broken link

**Comment by lreading** [comment]
Where `main` is the v2 codebase, do we just want to link to the `.com` tld?  I had updated it when we merged to main to point to the v2 docs.

**Comment by jgadsden** [comment]
Yes, that sounds good - if you could do the honours that would be great @lreading , and we can close the issue?
